style.css

```diff
@@ -0,0 +1,36 @@
+input {
+  display: block;
+}
+
+body {
+  background-color: rgb(132, 135, 130);
+}
+input {
+  border: 2px solid black
+  border-radius: 10px;
+  margin: 10px;
+  float: center;
+}
+button {
+  margin: 8px;
+}
+td {
+  color: white;
+}
+table tr td{
```

Comment:
No need to qualify your 'td' ruleset with `table tr td`, since `td` should always be a child  of them you can just write `td`.